stdlib/public/Platform/Platform.swift

```diff
@@ -351,11 +351,6 @@ public func ioctl(
 public func fork() -> Int32 {
   fatalError("unavailable function can't be called")
 }
-
-@available(*, unavailable, message: "Please use threads or posix_spawn*()")
-public func vfork() -> Int32 {
```

Comment:
Stupid question: are we sure `vfork` has the `returns_twice` attribute on all platforms?  It also seems unfortunate to lose this curated availability diagnostic for this API, although reducingthe size of the overlay is good.